Sources/Plasma/PubUtilLib/plAvatar/plArmatureEffects.h

```diff
@@ -109,7 +109,7 @@ class plArmatureEffectsMgr : public hsKeyedObject
         kMaxSurface,
         kFootNoSurface = kMaxSurface,
     };  
-    static const char *SurfaceStrings[];
+    static ST::string SurfaceStrings[];
```

Comment:
Technically, `ST::string` hasn't been forward declared yet in this header.

Reply:
True, but I we know it's available due to this being an `hsKeyedObject` subclass.